README.md

```diff
@@ -10,7 +10,7 @@
 `kotlinx-benchmark` is a toolkit for running benchmarks for multiplatform code written in Kotlin.
 It is designed to work with Kotlin/JVM, Kotlin/JS, Kotlin/Native, and Kotlin/Wasm (experimental) targets.
 
-To get started, ensure you're using Kotlin 1.9.20 or newer and Gradle 7.4 or newer.
+To get started, ensure you're using Kotlin 2.0.0 or newer and Gradle 7.4 or newer.
```

Comment:
Min supported version was bumped chiefly because of https://github.com/Kotlin/kotlinx-benchmark/issues/236